src/lib.rs

```diff
@@ -142,7 +142,7 @@ pub type BoxError = Box<dyn std::error::Error + Send + Sync + 'static>;
 /// - log_file: If defined, write output to a file starting with this name, ex app.log
 /// - log_level: error/warn/info/debug/trace, defaults to info
 /// - service_name:
-#[derive(Default, Clone, Debug)]
+#[derive(Default)]
```

Comment:
Might be desirable to add a custom Debug method, it is useful to find out what the config is.

As for clone, well, I suppose we can add a custom `clone_with_no_custom_layers`.  

Also, another way to solve the clone problem is to replace Box with Arc.

Reply:
Yeah, both these points make no sense.

I might have to raise a separate PR for including a custom filter for the default layer anyway, so I'll include these changes then.